web-client/src/components/SideTopMenu/SideTopMenu.tsx

```diff
@@ -13,7 +13,7 @@ import {
 import {
```

Comment:
Changed src/modules/myPosts to src/modules/myRequests.  Plan to have separate directory for myOffers.  Splitting out for understandability.